src/Ontology/Address.h

```diff
@@ -0,0 +1,56 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "PublicKey.h"
+
+#include <array>
+#include <vector>
+
+namespace TW {
+namespace Ontology {
+
+class Address {
+
+private:
+
+    std::vector<uint8_t> toScriptHash(std::vector<uint8_t> &data);
+
+public:
+
+    static const size_t size = 20;
+    static const uint8_t version = 0x17;
+
+    std::array<uint8_t, size> zero;
```

Comment:
zero?

Reply:
right, I follow the var name in Ontology source code, I think it will convenient to other developer to maintain these source code.

Reply:
I disagree, this should be called `bytes` or `data`

Reply:
I think `h160` is better. Install of `bytes` or `data `...

Reply:
Anyway, I will change var name to h160. and if you still not satisfied with this name, you can change it, the source code is under you control instead of me.

Reply:
You can in fact name it whatever you want. But please don't waste my time by posting a PR for which are not willing to receive feedback on.

Reply:
I receive feedback that really make sense, such as @hewigovens provide. In fact, you provide me some feedback that more like a personal preference, such as order alphabetically. however, the source code in your repository not be ordered alphabetically.

Reply:
https://github.com/TrustWallet/wallet-core/pull/103/commits/31af0e099566ea2fe7863a14d71107472e968c60